sources/packages/web/src/views/institution/locations/programs/LocationProgramAddEdit.vue

```diff
@@ -63,23 +85,23 @@ export default {
       return isAESTUser.value;
     });
     // Data-bind
-    const initialData = ref({});
+    const initialData = ref({} as any);
+    const institution = ref({} as InstitutionDetailAPIOutDTO);
```

Comment:
Why this was declared as a reactive variable?

Reply:
I needed to use it here 
<img width="359" alt="image" src="https://user-images.githubusercontent.com/72251620/168893768-ed705b12-45e1-49c5-a918-42ba26764f3e.png">

Reply:
@andrewsignori-aot  meant, that if a variable is not used inside vue template, we can use `const` or `let`. we need to use `ref` or `reactive` if the variable is accessed inside vue template like `initialData` here